### PR TITLE
Format tasks pract 4

### PR DESCRIPTION
**Pull request by plamena-ilieva**

### CODE REVIEW

**Review by dejansvetoslavov — APPROVED** [review]

File: SI/Sem.04/Pract.04/README.md [hunk 1]
```diff
@@ -4,51 +4,51 @@
 
 2. Да се напише функция isEven(int num), която връща дали дадено число е четно или не.
 
-3. Да се напише функция isPositive(int num), която връща дали дадено число е положително или не
+3. а) Да се напише функция isPositive(int num), която връща дали дадено число е положително или не
```
Comment (Justsvetoslavov) [inline SI/Sem.04/Pract.04/README.md:7]: Prefer to split this or if we combine to be without the a) and b).

**Review by Justsvetoslavov — CHANGES_REQUESTED** [review]
Prefer to either split tasks or if we combine them to be without a character list ( a) & b) ).

**Comment by dejansvetoslavov** [comment]
Looks good

File: SI/Sem.04/Pract.04/README.md [hunk 2]
```diff
@@ -4,51 +4,56 @@
...
+  
+8. Напишете функция pow(int num, unsigned N), която връща като резултат числото num повдигнато на степен N, където N е строго положително.
 
-11. Напишете функция calculate(int a, int b, char action), където action може да бъде [+-*/%].
+9. Напишете функция calculate(int a, int b, char action), където action може да бъде [+-*/%].
   Програмата да връща като резултат съответното действие, а ако не се укаже символ да върне сбора на числата.
 
-12. Напишете функция, askUser(int from, int to), която кара потребителя да въвежда числа докато не въведе число,
-  което да е в указания интервал, а като резултат връща въпросното число.
+10. Напишете програма, която по въведено число изкарва представянето му в двоична бройна система. <br />
```
Comment (GeorgiTerziev02) [inline SI/Sem.04/Pract.04/README.md:31]: They will learn binary on week 8

**Review by GeorgiTerziev02 — CHANGES_REQUESTED** [review]

**Review by GeorgiTerziev02 — APPROVED** [review]

File: SI/Sem.04/Pract.04/README.md (hunk 2 above)
Comment (Justsvetoslavov) [reply SI/Sem.04/Pract.04/README.md:31]: Bitwise operators are week 8.
Binary should be known or if not , we will address it.

**Review by Justsvetoslavov — APPROVED** [review]

**Pull request merged**